core/src/main/resources/hudson/model/Job/configure.properties

```diff
@@ -21,5 +21,5 @@
 # THE SOFTWARE.
 
 name={0} name
-Save=Save All
-Apply=Apply All
\ No newline at end of file
+Save=Save
+Apply=Apply
```

Comment:
Possibly a regression @gusreiber—I thought we had changed _Save_ to _Save All_ in 2.0 deliberately, as part of tabbed configuration pages.